spec/shell-session-spec.coffee

```diff
@@ -91,8 +94,23 @@ describe "Shell session grammar", ->
     expect(tokens[0][4].scopes).toEqual ['text.shell-session', 'support.function.builtin.shell']
 
   it "tokenizes shell output", ->
-    tokens = grammar.tokenizeLines('$ echo $FOO\nfoo')
+    tokens = grammar_session.tokenizeLines('$ echo $FOO\nfoo')
     temporaryScopeHack(tokens)
 
     expect(tokens[1][0].value).toBe 'foo'
     expect(tokens[1][0].scopes).toEqual ['text.shell-session', 'meta.output.shell-session']
+
+  it "", ->
+    tokens = grammar.tokenizeLines("${'root'}")
```

Comment:
It's not that I need it but it makes the test lighter:
If I use the old `grammar` (ie. `text.shell-session`) I need to add the `'text.shell-session` scope everywhere and a `> ` before `${'root'}`. I though this could be an occasion to add grammar since in the future most tests will be about `source.shell` and not so much about `text.shell-session`.